### ISSUE TITLE
Should South accept the invite?

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#945]
0F2D1D4C7B924FB4BA14F8EE7043
![image](https://github.com/user-attachments/assets/25ce554f-fab1-4be8-8660-bad13c7c7644)

Defining 2N as exact 10 HCP seems wrong.

But the range for Pass seems wrong as 17HCP should bid 3N, and even with 13 good like here 3N might be the right bid

Also interesting the 2N and 3N in North seems to have almost the same definition.

**Comment by EdwardPiwowar** [comment]
Defining 10 HCP after pass gives better results. But even if it is 11 S will pass without seeing any surpluses in partner.
But I think 2NT is a bad bid. N has more reason to bid 3N than S does after 2N.

**Comment by ThorvaldAagaard** [comment]
Yes, North was BBOBEN, and Pass in South could be OK (KnR 14.9 so it is a good hand)

The issue was raised because South could never have 17HCP and pass 2N - Generally we overcall light (so 13 HCP is not Min), so 2N seems right to me.

With 2 of 3 tops we might overcall 2D with 10-11 hcp depending on the hand
